runtime/src/chpl-mem-desc.c

```diff
@@ -34,16 +34,29 @@
 
 
 //
-// Define the description strings for the memory descriptors.
+// Define the description strings and track indicators for the memory
+// descriptors.
 //
-#define CHPL_MEMDESC_DESC(md_name, md_desc)  md_desc
-static const char* rt_memDescs[] = {
-  CHPL_MD_ALL_MEMDESCS(CHPL_MEMDESC_DESC)
+#define CHPL_MEMDESC_MACRO(_enum, _str, _track)  { _str, _track }
+static struct {
+  const char* string;
+  chpl_bool track;
+} rt_md[] = {
```

Comment:
This looks strange to me, should `rt_md[]...` be on a separate line with a declaration?